From d7fbd21c2ee1931fe3cf5e201801f82dec018473 Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Wed, 7 Dec 2016 17:50:05 -0600
Subject: [PATCH 201/208] crypto: allow testmgr to be skipped

---
 crypto/testmgr.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/crypto/testmgr.c b/crypto/testmgr.c
index 62dffa0028ac..197e9c530f1f 100644
--- a/crypto/testmgr.c
+++ b/crypto/testmgr.c
@@ -36,7 +36,7 @@
 
 #include "internal.h"
 
-static bool notests;
+static bool notests = true;
 module_param(notests, bool, 0644);
 MODULE_PARM_DESC(notests, "disable crypto self-tests");
 
-- 
2.11.0

