From 36e0aafb58a025e3192a636952f9d311709e5f7c Mon Sep 17 00:00:00 2001
From: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>
Date: Wed, 7 Dec 2016 17:53:45 -0600
Subject: [PATCH 202/208] silence "Power down" msg

---
 kernel/reboot.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/reboot.c b/kernel/reboot.c
index bd30a973fe94..eb6f8f6ee238 100644
--- a/kernel/reboot.c
+++ b/kernel/reboot.c
@@ -261,7 +261,7 @@ void kernel_power_off(void)
 		pm_power_off_prepare();
 	migrate_to_reboot_cpu();
 	syscore_shutdown();
-	pr_emerg("Power down\n");
+	pr_info("Power down\n");
 	kmsg_dump(KMSG_DUMP_POWEROFF);
 	machine_power_off();
 }
-- 
2.11.0

