From 47e08df6c7c5d81c6d87c673e68d35b835babc10 Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Tue, 29 Mar 2016 14:29:24 -0600
Subject: [PATCH 204/208] rdrand

---
 arch/x86/kernel/cpu/rdrand.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/x86/kernel/cpu/rdrand.c b/arch/x86/kernel/cpu/rdrand.c
index cfa97ff67bda..cd7aa166e3ba 100644
--- a/arch/x86/kernel/cpu/rdrand.c
+++ b/arch/x86/kernel/cpu/rdrand.c
@@ -48,6 +48,7 @@ void x86_init_rdrand(struct cpuinfo_x86 *c)
 	if (!cpu_has(c, X86_FEATURE_RDRAND))
 		return;
 
+	return;
 	for (i = 0; i < SANITY_CHECK_LOOPS; i++) {
 		if (!rdrand_long(&tmp)) {
 			clear_cpu_cap(c, X86_FEATURE_RDRAND);
-- 
2.11.0

