From 04bac7a91b44a2358ba05f1fe02cf8a4713ae4e1 Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Thu, 11 Feb 2016 11:06:26 -0600
Subject: [PATCH 205/208] reboot

---
 kernel/reboot.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/kernel/reboot.c b/kernel/reboot.c
index eb6f8f6ee238..b367c1e01a08 100644
--- a/kernel/reboot.c
+++ b/kernel/reboot.c
@@ -309,7 +309,7 @@ SYSCALL_DEFINE4(reboot, int, magic1, int, magic2, unsigned int, cmd,
 	 * halt when pm_power_off is not set do it the easy way.
 	 */
 	if ((cmd == LINUX_REBOOT_CMD_POWER_OFF) && !pm_power_off)
-		cmd = LINUX_REBOOT_CMD_HALT;
+		cmd = LINUX_REBOOT_CMD_RESTART;
 
 	mutex_lock(&reboot_mutex);
 	switch (cmd) {
@@ -326,7 +326,7 @@ SYSCALL_DEFINE4(reboot, int, magic1, int, magic2, unsigned int, cmd,
 		break;
 
 	case LINUX_REBOOT_CMD_HALT:
-		kernel_halt();
+		kernel_restart(NULL);
 		do_exit(0);
 		panic("cannot halt");
 
-- 
2.11.0

