From 8306e90af8936f6c02517ae7079182c65d121d76 Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Thu, 11 Feb 2016 11:07:54 -0600
Subject: [PATCH 206/208] no early modprobe

---
 kernel/kmod.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/kmod.c b/kernel/kmod.c
index bbbd977441c2..b70d6ae0e176 100644
--- a/kernel/kmod.c
+++ b/kernel/kmod.c
@@ -568,7 +568,7 @@ int call_usermodehelper_exec(struct subprocess_info *sub_info, int wait)
 	DECLARE_COMPLETION_ONSTACK(done);
 	int retval = 0;
 
-	if (!sub_info->path) {
+	if (!sub_info->path || system_state == SYSTEM_BOOTING) {
 		call_usermodehelper_freeinfo(sub_info);
 		return -EINVAL;
 	}
-- 
2.11.0

